spec/models/claim_spec.rb

```diff
@@ -35,27 +35,13 @@
     end
   end
 
-  context "that has a first name" do
-    it "validates the length of name is 100 characters or less" do
-      expect(build(:claim, first_name: "Name " * 50)).not_to be_valid
-      expect(build(:claim, first_name: "John")).to be_valid
-    end
-  end
-
   context "that has a middle name" do
     it "validates the length of middle name is 100 characters or less" do
       expect(build(:claim, middle_name: "Name " * 50)).not_to be_valid
       expect(build(:claim, middle_name: "Arnold")).to be_valid
     end
   end
 
-  context "that has a surname" do
-    it "validates the length of surname is 100 characters or less" do
-      expect(build(:claim, surname: "Name " * 50)).not_to be_valid
-      expect(build(:claim, surname: "Kimble")).to be_valid
-    end
-  end
```

Comment:
@martikat Might want to check with Andy if we want all names to now be consistent. Not a show stopper, more of a consistency question. Can fix under another ticket if required